.github/workflows/oadp-compatibility-check.yml

```diff
@@ -0,0 +1,46 @@
+name: OADP version compatibility check
+
+on:
+  push:
+    branches:
+      - master
+      # add release branches
+  schedule:
+    - cron: '0 0 * * 1' # Every Monday at 00:00 UTC
+  # TODO remove, just for test
```

Comment:
TODO

Reply:
After PR merge?

Reply:
before, just to test if the job works, but first need the oadp repo PR merged